alignment_comment.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
```

Comment:
If you want to debug and print every line to your terminal , you can use `set -x`